src/Tizen.NET.MaterialComponents/Components/MActionItem.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace Tizen.NET.MaterialComponents
+{
+    public class MActionItem
```

Comment:
Is there no method and event to activate the given Action? e.g. `Active()` and `Clicked`

Reply:
Fixed.